tools/build_api.py

```diff
@@ -23,6 +23,7 @@
 from os import linesep, remove, makedirs
 from time import time
 from intelhex import IntelHex
+from update import create_header
```

Comment:
Could we avoid this import? I don't want to have code in the tools that is specific to another product.